locator/tablet_sharder.hh

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "locator/tablets.hh"
+#include "dht/sharder.hh"
+#include "locator/token_metadata.hh"
+
+namespace locator {
+
+class tablet_sharder : public dht::sharder {
```

Comment:
tablets are an alternative to vnodes, I think we should try to avoid also making them participate in old-style sharding. I'll read on to see if I can find a way to avoid this.

Reply:
There are users which can stay uniform for vnodes and tablets, and I think there's value in that. If not dht::sharder, it would be something else.